cluster-autoscaler/cloudprovider/aws/aws_manager_test.go

```diff
@@ -244,3 +246,275 @@ func TestFetchAutoAsgs(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Empty(t, m.asgCache.get())
 }
+
+type ServiceDescriptor struct {
+	name                         string
+	region                       string
+	signingRegion, signingMethod string
+	signingName                  string
+}
+
+func TestOverridesActiveConfig(t *testing.T) {
+	tests := []struct {
+		name string
+
+		reader io.Reader
+		aws    provider_aws.Services
+
+		expectError        bool
+		active             bool
+		servicesOverridden []ServiceDescriptor
+	}{
+		{
+			"No overrides",
```

Comment:
For me readability and clarity would have these spelt out with their field names.

```Go
name: "foo',
expectError: false,
active: true,
// etc
...
```

but if these are as they are upstream so be it.

Reply:
Yes, I took it from upstream "as is".